docs/usage/insights.md

```diff
@@ -121,6 +121,35 @@ example-argocd-grpc         <none>   example-argocd-grpc         192.168.39.68
 example-argocd-prometheus   <none>   example-argocd-prometheus   192.168.39.68   80, 443   15h
 ```
 
+For OpenShift clusters, Routes will be created when route is enabled as shown in the below example.
+
+``` yaml
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+  labels:
+    example: insights
+spec:
+  grafana:
+    enabled: true
+    route:
+      enabled: true
+  prometheus:
+    enabled: true
+    route:
+      enabled: true
+  server:
+    insecure: true
+    route:
+      enabled: true
+```
+
+Initial Password for grafana is stored in the example-argocd-cluster secret. Password can be obtained from the secret by running the below command.
```

Comment:
Will this be the same password as used for Argo CD? Or does the secret have an additional key containing the Grafana password?

Reply:
its the same password @jannfis